resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1448 @@
+- slug: general
+  name: General descriptor schema
+  description: General descriptor schema.
+  version: 1.0.0
+  schema:
+    - name: general
+      label: General
+      group:
+        - name: name
+          label: Sample name
+          type: basic:string
+          required: false
+        - name: slug
+          label: Sample slug
+          type: basic:string
+          required: false
+        - name: id
+          label: Sample id
+          type: basic:integer
+          required: false
+        - name: collection
+          label: Collection
+          type: basic:string
+          required: false
+        - name: species
+          label: Species
+          type: basic:string
+          default: Homo sapiens
+          required: false
+          choices:
+            - label: Caenorhabditis elegans
+              value: Caenorhabditis elegans
+            - label: Cricetulus griseus
+              value: Cricetulus griseus
+            - label: Dictyostelium discoideum
+              value: Dictyostelium discoideum
+            - label: Dictyostelium purpureum
+              value: Dictyostelium purpureum
+            - label: Drosophila melanogaster
+              value: Drosophila melanogaster
+            - label: Homo sapiens
+              value: Homo sapiens
+            - label: Macaca mulatta
+              value: Macaca mulatta
+            - label: Mus musculus
+              value: Mus musculus
+            - label: Odocoileus virginianus texanus
```

Comment:
Remove `Odocoileus virginianus texanus` and `Solanum tuberosum` from the choices menu.